src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -336,8 +333,8 @@ function θ_surface_bc(
     prog_gm = center_prog_grid_mean(state)
     ρ_c = prog_gm.ρ
     kc_surf = kc_surface(grid)
-    ts_gm = center_aux_grid_mean_ts(state)
-    c_p = TD.cp_m(thermo_params, ts_gm[kc_surf])
+    ts_sfc = state.ts
+    c_p = TD.cp_m(thermo_params, ts_sfc)
```

Comment:
Why we don't need the kc_surf here anymore?

Reply:
`c_p` should be calculated from the thermo state at the surface instead of at `kc_surf = Cent(1)`, so instead of using `kc_surf` we just pass in the thermo state at the surface directly using `state.ts`

Reply:
So `state.ts` is the thermo state at the surface, not the thermo state of the entire column?

Reply:
Yes, it's the thermo state at the surface

Reply:
> Yes, it's the thermo state at the surface

Can we rename it to `ts_sfc` or something more clear?